test/test_connectionpool.py

```diff
@@ -557,22 +586,3 @@ def test_read_timeout_0_does_not_raise_bad_status_line_error(self) -> None:
                 timeout = Timeout(1, 1, 1)
                 with pytest.raises(ReadTimeoutError):
                     pool._make_request(conn, "", "", timeout)
-
-    def test_custom_http_response_class(self) -> None:
-        class CustomHTTPResponse(HTTPResponse):
-            pass
-
-        class CustomConnectionPool(HTTPConnectionPool):
-            ResponseCls = CustomHTTPResponse
-
-            def _make_request(self, *args: Any, **kwargs: Any) -> httplib.HTTPResponse:
-                httplib_response = httplib.HTTPResponse(MockSock)  # type: ignore[arg-type]
-                httplib_response.fp = MockChunkedEncodingResponse([b"f", b"o", b"o"])  # type: ignore[assignment]
-                httplib_response.headers = httplib_response.msg = httplib.HTTPMessage()
-                return httplib_response
-
-        with CustomConnectionPool(host="localhost", maxsize=1, block=True) as pool:
-            response = pool.request(
-                "GET", "/", retries=False, chunked=True, preload_content=False
-            )
-            assert isinstance(response, CustomHTTPResponse)
```

Comment:
The most recent changes that were asked for removed this functionality. So I removed this failing test.